packages/ketcher-core/src/application/editor/operations/monomer/index.ts

```diff
@@ -48,6 +48,19 @@ export class MonomerHoverOperation implements Operation {
     );
   }
 }
+export class APHoverOperation implements Operation {
```

Comment:
I propose to rename `AP` to attachmentPoint everywhere.

Reply:
Replaced